PowerPointLabs/PowerPointLabs/FrameMotionAnimation.cs

```diff
@@ -157,5 +177,80 @@ private static void AddFrameAnimationEffects(PowerPointSlide animationSlide, Pow
                 disappearLast.Timing.TriggerDelayTime = duration;
             }
         }
+
+        private static PowerPoint.Shape MakePivotCenteredLine(PowerPointSlide animationSlide, PowerPoint.Shape line)
+        {
+            // Add a 180 degree rotated line to offset its pivot shift caused by arrowhead
+            PowerPoint.Shape transparentLine = line.Duplicate()[1];
+            transparentLine.Line.Transparency = 1.0f;
+            transparentLine.Rotation = 180.0f;
+            transparentLine.Left = line.Left;
+            transparentLine.Top = line.Top;
+
+            PowerPoint.Shape tempAnimationHolder = line.Duplicate()[1];
+            animationSlide.TransferAnimation(line, tempAnimationHolder);
+
+            List<PowerPoint.Shape> toGroup = new List<PowerPoint.Shape> { line, transparentLine };
+            PowerPoint.Shape newLine = animationSlide.ToShapeRange(toGroup).Group();
+
+            animationSlide.TransferAnimation(tempAnimationHolder, newLine);
+            tempAnimationHolder.Delete();
+
+            return newLine;
+        }
+
+        private static double GetLineAngle(PowerPoint.Shape shape, bool acute = false)

```

Comment:
Its supposedly a flag on whether the method should return the angle as an acute angle, turns out i can reduce the number of trigo functions and its not needed anymore (removed in latest commit).